openmetadata-docs/content/v1.6.x-SNAPSHOT/connectors/database/glue/index.md

```diff
@@ -7,8 +7,8 @@ slug: /connectors/database/glue
 name="Glue"
 stage="PROD"
 platform="OpenMetadata"
-availableFeatures=["Metadata", "dbt"]
-unavailableFeatures=["Query Usage", "Owners", "Tags", "Stored Procedures", "Data Profiler", "Data Quality", "Lineage", "Column-level Lineage"]
+availableFeatures=["Metadata", "dbt", "Lineage"]
```

Comment:
```suggestion
availableFeatures=["Metadata", "dbt", "External Table Lineage"]
```


just to avoid the confusion that it supports query or any other type of lineage